include/deal.II/lac/petsc_matrix_free.h

```diff
@@ -14,15 +14,15 @@
 // ---------------------------------------------------------------------
 
 #ifndef dealii_petsc_matrix_free_h
-#  define dealii_petsc_matrix_free_h
+#define dealii_petsc_matrix_free_h
 
 
-#  include <deal.II/base/config.h>
+#include <deal.II/base/config.h>
 
-#  ifdef DEAL_II_WITH_PETSC
-#    include <deal.II/lac/exceptions.h>
-#    include <deal.II/lac/petsc_matrix_base.h>
-#    include <deal.II/lac/petsc_vector.h>
+#ifdef DEAL_II_WITH_PETSC
+#  include <deal.II/lac/exceptions.h>
+#  include <deal.II/lac/petsc_matrix_base.h>
+#  include <deal.II/lac/petsc_vector.h>
```

Comment:
I'd still like to get rid of this kind of indentation but that's outside the scope of this PR.

Reply:
That would be nice!